app/assets/stylesheets/application.scss

```diff
@@ -76,3 +75,17 @@
     margin: 0;
   }
 }
+
+#map-container {
+   height: 400px;
+   border-radius: 16px 16px;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Shorthand form for property `border-radius` should be written more concisely as `16px` instead of `16px 16px`